packages/atomic/src/pages/index.html

```diff
@@ -237,7 +237,7 @@
       <atomic-search-box></atomic-search-box>
       <atomic-facet-manager>
         <atomic-category-facet field="geographicalhierarchy" label="World Atlas" with-search> </atomic-category-facet>
-        <atomic-facet field="author" label="Authors"></atomic-facet>
+        <atomic-facet field="author" label="Authors" custom-sort="BillV"></atomic-facet>
```

Comment:
What do you think of the option name? How would you compare it to `prioritized-values`?

Reply:
To be honest, I'm not sure I would add anything in atomic for this right now...

Was that a specific request from someone ?

Because my gut felling tells me that what end users would actually be interested in is "the whole shebang", which means being able to curate data requested and returned from the API, not merely being able to order a list of values client side.

Similar to what we had in JSUI for the old facet. 

There is some value in adding a custom sort client side, but then I guess the name "custom-sort" would be misleading, as your comment indicate. 

And that if/when the API adds more support for custom sort, we would be introducing a breaking change for this option, since it would have a different behaviour. 

Thoughts ?

Reply:
I agree, the option is not great as-is. It might work well in one scenario; a field with a small number of values.
Aside from this, I'm not so sure,

> Was that a specific request from someone ?

@ThibodeauJF maybe you can give us more context? I see you created the jira recently, in June 2021.


Personally,
- If sorting client-side is ok, then this is possible with headless. One can also create a field with a small number of values to increase the chance of values appearing in the response.
- If this is not good enough, and it is a high priority for customers, as you said Olivier, we'll need to collaborate with the search api team.

Reply:
I proposed it in the Facets V1 design document IIRC when looking at what we should provide with them. Custom sort was not available so I entered a new Jira in the backlog.

That kind of functionality, and "allowedValues", is often used/requested by clients. Let's say you have an ecommerce page want to display bed sizes, single, double, queen, king, you don't want the index/ml order there, but the logical order.

I would push harshly for API support through headless